tests/contrib/kafka/test_kafka.py

```diff
@@ -129,18 +129,21 @@ def non_auto_commit_consumer(tracer, kafka_topic):
 
 @pytest.fixture
 def serializing_producer(tracer):
-    _producer = confluent_kafka.Producer({"bootstrap.servers": BOOTSTRAP_SERVERS})
+    _producer = confluent_kafka.SerializingProducer(
+        {"bootstrap.servers": BOOTSTRAP_SERVERS, "value.serializer": lambda x, y: x}
```

Comment:
What's value.serializer? It looks like a lambda that just returns the first value it's called with?

Reply:
That's right. It's a function that gets called with each message as an argument and that returns the message in a serialized form. https://docs.confluent.io/platform/current/clients/app-development.html

Reply:
Ahhh got it, thanks!